LiveSplit.AutoSplitters.xml

```diff
@@ -1,5 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <AutoSplitters>
+    <AutoSplitter>
+        <Games>
+            <Game>We Were Here Too</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/Zeuba-Speedruns/AutoSplitters/main/We%20Were%20Here%20Too/WeWereHereToo.asl</URL>
+            <URL>https://github.com/just-ero/asl-help/raw/main/lib/asl-help</URL>
```

Comment:
I would suggest pinning this to the commit you tested `asl-help` on. It would ensure your autosplitter doesn't break when there's an update.